projects/packages/external-media/src/features/admin/external-media-import.js

```diff
@@ -0,0 +1,90 @@
+import { sprintf, _n } from '@wordpress/i18n';
+import { useState, useEffect } from 'react';
+import ReactDOM from 'react-dom/client';
+import { getExternalLibrary } from '../../shared';
+
+const JETPACK_EXTERNAL_MEDIA_IMPORT_PAGE_CONTAINER = 'jetpack-external-media-import';
+const JETPACK_EXTERNAL_MEDIA_IMPORT_PAGE_MODAL = 'jetpack-external-media-import-modal';
+
+const JetpackExternalMediaImport = () => {
+	const [ selectedSource, setSelectedSource ] = useState( null );
+	const ExternalLibrary = getExternalLibrary( selectedSource );
+
+	const showNotice = message => {
+		const notice = document.createElement( 'div' );
+		notice.className = 'notice notice-success';
+		notice.innerHTML = `<p>${ message }</p>`;
+
+		// Add the success notice after the page title
+		const heading = document.querySelector(
+			`#${ JETPACK_EXTERNAL_MEDIA_IMPORT_PAGE_CONTAINER } > h1`
+		);
+		if ( heading ) {
+			heading.parentNode.insertBefore( notice, heading.nextSibling );
+		}
+	};
+
+	const handleSelect = media => {
+		if ( ! media || media.length === 0 ) {
+			return;
+		}
+
+		showNotice(
+			sprintf(
+				/* translators: %d is the number of the media file */
+				_n(
+					'%d media file imported successfully',
```

Comment:
Nit: I think it would be better to end the sentence with period (`.`) to be consistent with all other notices in the page 

<img width="369" alt="image" src="https://github.com/user-attachments/assets/8a188475-33a8-4fd1-b16b-51105e42cd07" />